docs/MIGRATION_V1_V2.md

```diff
@@ -0,0 +1,80 @@
+| [E024](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E024) 	| [SameNameAndDescriptionForRouteNotice](/RULES.md#SameNameAndDescriptionForRouteNotice)                                                                                                                                                    	| [ERROR](/RULES.md#errors)     	|
+| [E025](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E025) 	| [RouteColorContrastNotice](/RULES.md#RouteColorContrastNotice)                                                                                                                                                                            	| [ERROR](/RULES.md#errors)     	|
+| [E026](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E026) 	| [UnexpectedEnumValueNotice](/RULES.md#UnexpectedEnumValueNotice)                                                                                                                                                                          	| [WARNING](/RULES.md#warnings) 	|
+| [E027](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E027) 	| [RouteBothShortAndLongNameMissingNotice](/RULES.md#RouteBothShortAndLongNameMissingNotice)                                                                                                                                                	| [ERROR](/RULES.md#errors)     	|
+| [E028](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E028) 	| [RouteShortAndLongNameEqualNotice](/RULES.md#RouteShortAndLongNameEqualNotice)                                                                                                                                                            	| [WARNING](/RULES.md#warnings) 	|
+| [E029](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E029) 	| [MissingRequiredFieldError](/RULES.md#MissingRequiredFieldError)                                                                                                                                                                          	| [ERROR](/RULES.md#errors)     	|
+| [E030](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E030) 	| [InconsistentAgencyTimezoneNotice](/RULES.md#InconsistentAgencyTimezoneNotice)                                                                                                                                                            	| [ERROR](/RULES.md#errors)     	|
+| [E031](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E031) 	| [Not implemented](https://github.com/MobilityData/gtfs-validator/pull/799)                                                                                                                                                                	| N/A                              	|
```

Comment:
To make it easier to see what wasn't implemented, I would leave the "Not implemented" text without a hyperlink, and then next to it add "(see pull request XXXXX)", with "pull request XXXXX" being hyperlinked.